src/6_branch.js

```diff
@@ -839,10 +839,13 @@ Branch.prototype['sendSMS'] = wrap(
 						return done(err);
 					}
 					var url = data['url'];
+					if (!/(bnc.lt\/|app\.link\/)/.test(url)) {
+						url = config.link_service_endpoint + utils.extractDeeplinkPath(url);
+					}
```

Comment:
If the url returned from the link service is already on a Branch domain, just use it...otherwise, set it to be a Branch domain (this is just-in-case, and may never be needed)